modules/sdk-coin-avaxp/src/lib/transaction.ts

```diff
@@ -177,7 +177,7 @@ export class Transaction extends BaseTransaction {
       type: this.type,
       memo: utils.bufferToString(this.avaxPTransaction.getMemo()),
       signatures: this.signature,
-      stakedOutputs: this.stakedOutputs,
+      outputs: this.outputs,
```

Comment:
backward incompatible changes will need to be `BREAKING` changes. see https://github.com/BitGo/BitGoJS/pull/2565

Reply:
done

Reply:
Note, when we release, this will be a breaking change internally as users switch between the names, but not for the stable release so this won't bump a major version. See:

https://github.com/BitGo/BitGoJS/blob/rel/latest/modules/sdk-coin-avaxp/src/lib/iface.ts#L26
https://github.com/BitGo/BitGoJS/blob/rel/latest/modules/sdk-coin-avaxp/src/lib/transaction.ts#L178

Reply:
Since this coin is in limited integration, so there are few users.
Explain Transaction hasn't break changes and is used in SDK and WP. 
Even so, we will watch over the dump the sdk.